tests/slow/src/test/scala/tests/BloopPantsSuite.scala

```diff
@@ -11,6 +11,8 @@ class BloopPantsSuite extends FastpassSuite {
                            |/pants.ini
                            |[GLOBAL]
                            |pants_version: 1.26.0.dev0
+                           |[resolver]
+                           |resolver: coursier
```

Comment:
TIL

Reply:
Yes, it was surprising to me as well. I don't know why Ivy ignores the flag...